jobtastic/__init__.py

```diff
@@ -14,4 +14,4 @@
 
 # -eof meta-
 
-from jobtastic.task import JobtasticTask
+from jobtastic.task import JobtasticTask  # NOQA
```

Comment:
It is in `__all__`. The NOQA is because it's a module import that's not at the top of the file. It's not at the top because we use string parsing to get the VERSION and other project information for setup.py.